.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,77 @@
+# Release vFIXME
+
+## Checklist
+
+- [ ] [Create a new github milestone](https://github.com/DataDog/dd-trace-go/milestones/new) for the next version and
+  move any open PRs and issues that are in the current release milestone to the new one.
```

Comment:
Moving all of the open PRs and issues should probably be done after the release. Otherwise, I'm imagining a scenario where you move them all at 8am, do the release at 4pm, and sometime between 8am-4pm someone has opened a new issue/PR and put it in the wrong milestone.

Reply:
Indeed. Now fixed and moved at the bottom of the release process, after the release was published.